libraries/USB/src/USBHID.cpp

```diff
@@ -340,8 +340,10 @@ bool USBHID::SendReport(uint8_t id, const void* data, size_t len, uint32_t timeo
             }
         }
     }
-
     xSemaphoreGive(tinyusb_hid_device_input_mutex);
+    if (tinyusb_hid_device_input_sem) {
```

Comment:
I think it is not needed to check it, since it will crash anyway in this code if semaphore is NULL
```
            xSemaphoreTake(tinyusb_hid_device_input_sem, 0);
            if(xSemaphoreTake(tinyusb_hid_device_input_sem, timeout_ms / portTICK_PERIOD_MS) != pdTRUE){
            ```

Reply:
makes sense, check removed